sdks/java/harness/src/main/java/org/apache/beam/fn/harness/FnHarness.java

```diff
@@ -107,6 +111,29 @@ private static Endpoints.ApiServiceDescriptor getApiServiceDescriptor(String des
     return apiServiceDescriptorBuilder.build();
   }
 
+  public static String removeNestedKey(String jsonString, String keyToRemove) throws Exception {
```

Comment:
Not sure when we can put this json util function. Any suggestion is welcome.

Reply:
You could just put it in `org.apache.beam.sdk.util` package somewhere. We have a ton of random junk there.

Reply:
or leave it here until you need to move it, better

Reply:
Yeah, leave it is fine. Seemingly it introduced a new dependency jackson_databind, but since this is also a dependency of java_core, it does not matter